custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
+  text: `Bringing exploratory mindset to technical leadership,
+        with six years of experience in software development and computational research`,
 };
 
 const projects = [
   {
-    title: faker.lorem.words(2),
-    text: faker.lorem.words(5),
+    title: "Software development",
+    text: "Full stack development description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
```

Comment:
Use descriptive variable names instead of random text generation for project titles.